packages/core/src/geometry/GeometrySystem.js

```diff
@@ -42,6 +42,11 @@ export default class GeometrySystem extends System
          * @private
          */
         this.cache = {};
+
+        /**
+         * A cache of currently bound buffer..
```

Comment:
Could you add at `@member {object}` to this doc? Also would be nice to elaborate on the description here if this is, in fact, public.